core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -248,14 +256,18 @@ public void onNext(HttpObject o) {
         ctx.flush();
     }
 
+    private boolean isHttp1() {
+        return responseEncoder instanceof Http1ObjectEncoder;
```

Comment:
Can we get this information from `reqCtx.sessionProtocol()`? If not, could you add some comments on why?

Reply:
> Can we get this information from reqCtx.sessionProtocol()?

Yes, we can. 😄